django/utils/cache.py

```diff
@@ -180,10 +180,13 @@ def get_conditional_response(request, etag=None, last_modified=None, response=No
             return _precondition_failed(request)
 
     # Step 4: Test the If-Modified-Since precondition.
-    if (not if_none_match_etags and if_modified_since and
-            not _if_modified_since_passes(last_modified, if_modified_since)):
-        if request.method in ('GET', 'HEAD'):
-            return _not_modified(request, response)
+    if (
+        not if_none_match_etags and
+        if_modified_since and
```

Comment:
PEP 8 hedges about line breaks and binary operators but suggests ultimately that breaking before the operator is better. As long as we're touching this I would suggest breaking before the operators.

Reply:
In Django we break after the operator, that's why `W504` is ignored. It's checked by `flake8`.